contracts/modules/capital/Pool.sol

```diff
@@ -19,7 +21,8 @@ contract Pool is IPool, MasterAware, ReentrancyGuard {
   uint16 constant MAX_SLIPPAGE_DENOMINATOR = 10000;
 
   /* storage */
-  Asset[] public override assets;
+  Asset[] public override payoutAssets;
```

Comment:
This array is used for two purposes:
- Detect if the user can pay the cover premium using this asset
- Use this asset in a payout
The current naming only suggest the later and fails to signal the first one, for this reason I suggest using the `coverAssets` name instead.